docs/src/dedicated/overview/grid.md

```diff
@@ -28,46 +49,54 @@ PHP options that can we can change via support ticket include:
+a cluster of Solr servers to ensure high availability.
+This shouldn't affect you most of the time, but may influence certain advanced use cases.
 
-Specifically, when a deploy to either Production or Staging begins, any active cron tasks are sent a `SIGTERM` message so that they can terminate gracefully if needed.  If they are still running 2 seconds later a `SIGKILL` message will be sent to forcibly terminate the process.
+## Cron tasks interrupted by deploys
 
-For that reason, it's best to ensure your cron tasks can receive a SIGTERM message and terminate gracefully.
+How [cron tasks](../../create-apps/app-reference.md#crons) interact with deploys changes based on the environment.
 
-## Configuration & change management
-
-Some configuration parameters for Dedicated clusters cannot be managed via the YAML configuration files, and for those parameters you will need to open a support ticket to have the change applied.  Further, the `.platform/routes.yaml` and `.platform/services.yaml` files do not automatically apply.  Those will apply on the development environments but not on the staging and production instances.  Any existing service upgrades or new service additions to staging and production will require a support ticket.
+On Grid environments, a running cron task blocks a deploy until the cron is complete.
+On Dedicated environments, a deploy terminates a running cron task.
 
-It is possible to run different configurations for some (but not all) options between staging and production, such as cron tasks.  By default we will make configuration changes to both instances unless you request otherwise.
+Specifically, when a deploy to either Production or Staging begins,
+any active cron tasks are sent a `SIGTERM` message so that they can terminate gracefully.
+If they're still running 2 seconds later, a `SIGKILL` message is sent to forcibly terminate the process.
 
-Specifically:
+So it's best to ensure your cron tasks can receive a `SIGTERM` message and terminate gracefully.
 
-* Cron commands
-* Worker instances
-* Service versions and configuration (everything in `.platform/services.yaml`)
-* Route, domain, and redirect configuration (everything in `.platform/routes.yaml`)
-* Application container version
-* Additional PHP extensions
-* Web server configuration (the web.locations section of `.platform.app.yaml`)
+## Configuration & change management
 
-### Cron
+You can't manage some configuration settings via YAML configuration files on Dedicated environments.
+In these cases, you need to open a support ticket.
+You can have some settings different between Staging and Production environments.
+It's assumed you want the settings the same, unless you state otherwise in the ticket.
 
-Cron tasks may run up to once per minute (They are limited to once every 5 minutes on Platform.sh Professional plans).
+The following settings require a support ticket:
```

Comment:
:hammer_and_wrench: ⏯️ 
Since they might want direct action, dedicated customers don't need their TAMs for that, right?
```suggestion
The following settings require [a support ticket](https://console.platform.sh/-/users/~/tickets/open):
```